src/env.h

```diff
@@ -608,6 +608,13 @@ class Environment {
 
   std::unordered_multimap<int, loader::ModuleWrap*> module_map;
 
+  struct PackageConfig {
+    bool exists;
+    bool has_main;
+    std::string main;
+  };
```

Comment:
PackageConfig should probably live in the `node::loader` namespace, like ModuleWrap.